src/main/java/com/cloudbees/jenkins/Cleaner.java

```diff
@@ -2,6 +2,7 @@
 
 import hudson.Extension;
 import hudson.model.AbstractProject;
```

Comment:
BTW you could clean up this import (if you fixed the obsolete Javadoc link in line 32).